### PR TITLE
Refactor/declarative outlier app enhancement

### PR DESCRIPTION
**Pull request by nayan2167**
outliers_declatative.md has been updated with similar changes made in #5302 

The file can be previewed [here](https://github.com/nayan2167/panel/blob/refactor/declarative-outlier-app-enhacements/doc/explanation/api/examples/outliers_declarative.md)

### CODE REVIEW

File: doc/explanation/api/examples/outliers_declarative.md [hunk 1]
```diff
@@ -23,40 +22,51 @@ data = pd.read_csv(csv_file, parse_dates=['date'], index_col='date')
...
+    return avg.hvplot(height=300, width=400, legend=False) * highlight.hvplot.scatter(
+        color="orange", padding=0.1, legend=False
+    )
 ```
 
-Now, let's implement the declarative API approach:
+By applying specific parameters to the `create_plot` function, we can generate a plot with a unique dataset.
 
 ```{pyodide}
-class RoomOccupancy(param.Parameterized):
```
[the remark this answers is not in the document]
Comment (nayan2167) [reply doc/explanation/api/examples/outliers_declarative.md:43]: Understood 👍

[the remark this answers is not in the document]
Comment (nayan2167) [reply doc/explanation/api/examples/outliers_declarative.md]: @MarcSkovMadsen I made a few changes please review it.

[the remark this answers is not in the document]
Comment (MarcSkovMadsen) [reply doc/explanation/api/examples/outliers_declarative.md]: Looks great to me now

**Comment by philippjfr** [comment]
Very sorry that this never got merged, similar changes have been applied since so I'll close.

**Pull request closed without merging**